Registry.Dockerfile

```diff
@@ -22,4 +22,6 @@ COPY --from=build /app/publish .
 EXPOSE 5000
 EXPOSE 5001
 
+ENV ReturnComittedForFinalized=true
```

Comment:
When you have time, can you then explain why this is an configuration? According to your PR description it is to ensure backwards compatibility in some way, but I am unable to figure out if it is really needed.

Reply:
For a couple of reasons
- First to enable people to override it and disable the backward compatibility.
- Second ensure that all tests can be done with the new Finalized state. 
- And it is defaulted both in the chart and containers, the container specifically because it is used in tests in the other components, and would otherwise fail until they are moved to the new spec.